### ISSUE TITLE
Prepend docz CSS classnames to prevent conflicts

### ISSUE THREAD

**Issue by auermi** [doczjs/docz#1271]
We have a component that uses the className `tag`. This is causing conflicts with elements with className of tag used in the Playground.

A potential solution would be to prepend all docz CSS classes with `docz-`

Example:
![Screen Shot 2019-11-20 at 10 01 12 AM](https://user-images.githubusercontent.com/5431160/69249914-b7757580-0b7c-11ea-98a3-4d96000103b5.png)

**Comment by rakannimer** [comment]
Hey @auermi 

Thanks for reporting this issue. 

This seems [prism-react-renderer](https://github.com/FormidableLabs/prism-react-renderer) specific. 

We can either open an issue there with a request to allow us to prepend some string to the class names or wrap the playground editor in an iframe.

**Comment by rakannimer** [comment]
Hey @auermi  

This is solved in 2.2.0. 

To scope your playground styles and make them independent of docz theme css you have 2 choices : 

1. add a useScoping prop to your playground

```jsx
<Playground useScoping> 
{
// blablabla
}
</Playground>
```
2. add `useScopingInPlayground : true` in doczrc.js 

```js
export default {
	useScopingInPlayground: true
}
```

Sorry for the delay in the reply but the fix was implemented thanks to you raising this issue ! So thanks 👍 

Going to close it, please re-open or comment if for some reason something doesn't work for you !

If you're interested in implementation details check #1305

**Comment by auermi** [comment]
@rakannimer Thanks!
